mizrahi2021re/openreview/packages.tex

```diff
@@ -0,0 +1,47 @@
+ % if you need to pass options to natbib, use, e.g.:
+%     \PassOptionsToPackage{numbers, compress}{natbib}
+% before loading neurips_2019
+
+% ready for submission
+% \usepackage{neurips_2019}
+
+% to compile a preprint version, e.g., for submission to arXiv, add add the
+% [preprint] option:
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+%\usepackage[final]{neurips_2019} % Remove for ReScience
+
+% to avoid loading the natbib package, add option nonatbib:
+% \usepackage[nonatbib, final]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage[dvipsnames]{xcolor}
```

Comment:
From the logs, it seems this import is the issue. Can you try removing this package?

(https://github.com/dmizr/RC-2020/blob/main/mizrahi2021re/journal/article.log#L1360)

Reply:
Thank you! This import was definitely problematic and the locally compiled version looks much better now.
However, there still seems to be a few issues remaining, such as the font not matching the one used in other ReScience articles.
- New compiled PDF can be found here: https://github.com/ReScience/RC-2020/blob/25d68c6c427676b32845527f9a269cee7cd9375b/mizrahi2021re/journal/article.pdf
- New log: https://github.com/ReScience/RC-2020/blob/25d68c6c427676b32845527f9a269cee7cd9375b/mizrahi2021re/journal/article.log

Reply:
I think this is good enough! We can inspect the font mismatch on our end!